sigar_linux_common_test.go

```diff
@@ -0,0 +1,89 @@
+//go:build linux
+// +build linux
+
+package gosigar_test
+
+import (
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	sigar "github.com/elastic/gosigar"
+	"github.com/stretchr/testify/assert"
+)
+
+func setUpCommonTest(t testing.TB) {
+	oldMtabf := sigar.Mtabf
+	mtabDir := t.TempDir()
+	sigar.Mtabf = filepath.Join(mtabDir, "mtab")
```

Comment:
Nit. Variable not needed.
```suggestion
	sigar.Mtabf = filepath.Join(t.TempDir(), "mtab")
```